client/src/MovementDispatcher.h

```diff
@@ -0,0 +1,29 @@
+//
+// Created by tdancois on 1/14/17.
+//
+
+#ifndef ARES_MOVEMENTDISPATCHER_H
+#define ARES_MOVEMENTDISPATCHER_H
+
+#include <memory>
+#include "PlayerCommands.h"
+#include "NetworkThread.h"
+
+class MovementDispatcher {
+ private:
+  const int32_t refreshRateMs;
+  sf::Clock clock;
+  bool needUpdate;
+
+ public:
+  explicit MovementDispatcher(const int32_t refreshRateMs = 50);
```

Comment:
You can set the refresh rate at 1s

It's not necessary to overload th network now.